doc/source/ray-metrics.rst

```diff
@@ -0,0 +1,132 @@
+Monitoring Ray
+==============
+To help monitoring Ray applications, Ray
+
+- Collects Ray's pre-selected system level metrics.
+- Exposes metrics through Prometheus endpoints.
+- Supports application-level metrics collection APIs (EXPERIMENTAL).
+
+.. note::
+
+    It is currently an experimental feature and under active development. APIs are subject to change.
+
+Getting Started (Single Node)
```

Comment:
Can we just drop the `ray.init()` documentation for now? 

The UX seems quite odd (it seems like you're exposing an internal configuration via the underscore, and it doesn't seem like a common use case).